src/components/Header/Header.tsx

```diff
@@ -0,0 +1,54 @@
+'use client'
+
+import { header_style } from '@/styles/header-style'
+import { BottomNavigation, BottomNavigationAction, Box } from '@mui/material'
+import React from 'react'
+
+interface Props {}
+
+function Header({}: Props) {
+  const [value, setValue] = React.useState(0)
+  return (
+    <Box
+      component={'header'}
+      sx={{
+        width: '100vw',
+        display: 'flex',
+        justifyContent: 'center',
+        borderBottom: 2,
+        paddingY: '1.2rem',
+        borderColor: '#e7e5e4',
+      }}
+    >
+      <Box sx={{ display: 'flex', flexDirection: 'column', rowGap: '0.4rem' }}>
+        <Box
+          sx={{
+            fontSize: 28,
+            fontWeight: '800',
+          }}
+        >
+          OpenBC
+        </Box>
+
+        <BottomNavigation
+          showLabels
+          value={value}
+          onChange={(event, newValue) => {
+            if (typeof newValue === 'number') setValue(newValue)
```

Comment:
```suggestion
          onChange={(event, newValue: number) => {
            setValue(newValue);
```
Tested in my instance of VSCode, no errors on my side:

![image](https://github.com/OpenBCca/openbc-web/assets/16601729/1ca796ef-a09f-41bc-97fc-5caa65a410c2)

Reply:
Do you have this extension installed?
![image](https://github.com/OpenBCca/openbc-web/assets/72838071/45e3e1ec-79dc-465e-98b9-5b02f3d7f1e4)

Reply:
Ahh thanks for the reminder for installing ESLint, I didn't have it on the machine I was doing the review on it.

With it installed, I see:

![image](https://github.com/OpenBCca/openbc-web/assets/16601729/f0b3941e-0aac-43b1-a053-a05deff4f30c)

But once I declare `newValue` as a `number`, I don't seem to get the error above any more:

![image](https://github.com/OpenBCca/openbc-web/assets/16601729/eb738aac-3bbd-4032-967e-d2ec4cd3b134)

But if we prefer the current implementation, or if perhaps I need to configure ESLint differently, then I'm happy to just leave this as-is!

Reply:
yes the newValue:number works too